newsfragments/45327.significant.rst

```diff
@@ -0,0 +1 @@
+Renamed DAG Property ``fail_stop`` to ``fail_fast`` across the codebase to align with Airflow 3.0.
```

Comment:
Its not exactly to align with Airflow 3.0, but to make it semantically more correct to serve the purpose the parameter suggests.

Reply:
@hprassad  Please use this template to create newsfragment. https://github.com/apache/airflow/blob/main/newsfragments/template.significant.rst
 Thanks!